src/hooks/UrqlUseQuery.rei

```diff
@@ -7,4 +7,16 @@ type useQueryState('a) = {
 }
 type useQueryResponse('a) = (useQueryState('a), partialOperationContextFn);
 
-let useQuery: (~query: string, ~variables: 'a=?, ~requestPolicy: UrqlTypes.requestPolicy=?, ~pause: bool=?, unit) => useQueryResponse('b)
+let useQuery:
+  (
+    ~request: {
+                .
+                "parse": Js.Json.t => 'response,
+                "query": string,
+                "variables": 'vars,
```

Comment:
AFAIK the type of `variables` returned by calling make is always a `Js.Json.t`: https://github.com/mhallin/graphql_ppx#send-queries. Any reason to make it a type arg here rather than just specifying `Js.Json.t`?